helm/nginx-ingress-controller-app/templates/controller-service.yaml

```diff
@@ -15,7 +15,8 @@ metadata:
   {{- if eq .Values.controller.service.type "LoadBalancer" }}
   {{- if eq .Values.provider "aws" }}
     {{- if not .Values.controller.service.public }}
-    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+    service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+    service.beta.kubernetes.io/aws-load-balancer-scheme: "internal"
```

Comment:
Shouldn't this be `internet-facing`, since service is public? But don't know any details about all this.

Reply:
ah, "not public" ;) ok..